charts/langsmith/scripts/generate_clickhouse_stats.sh

```diff
@@ -0,0 +1,165 @@
+#!/bin/sh
+
+# Function to generate a fake UUID in lowercase
+generate_uuid() {
+    if command -v uuidgen >/dev/null 2>&1; then
+        uuidgen | tr '[:upper:]' '[:lower:]'
+    else
+        echo "uuidgen command not found. Exiting..."
+        exit 1
+    fi
+}
+
+# Argument Parsing
+clickhouse_url=""
+debug=""
+ssl=""
+days=14
+interval=1
+output_file=""
+cluster=""
+cluster_flag_passed=false
+
+while [ $# -gt 0 ]; do
+    case "$1" in
+        --debug)
+            debug="--debug"
+            shift
+            ;;
+        --ssl)
+            ssl="--ssl"
+            shift
+            ;;
+        --days)
+            days="$2"
+            shift 2
+            ;;
+        --interval)
+            interval="$2"
+            shift 2
+            ;;
+        --output)
+            output_file="$2"
+            shift 2
+            ;;
+        --cluster)
+            cluster="true"
+            cluster_flag_passed=true
+            shift
+            ;;
+        *)
+            if [ -z "$clickhouse_url" ]; then
+                clickhouse_url="$1"
+                shift
+            else
+                echo "Unknown argument: $1"
+                echo "Usage: $0 <clickhouse_url> [--debug] [--ssl] [--days <number_of_days>] [--interval <number_of_hours>] [--output <path/to/file.csv>] [--cluster]"
```

Comment:
maybe worth moving this and the 2 lines below to a `print_usage_and_exit()` fn for reuse